networkx/algorithms/flow/maxflow.py

```diff
@@ -13,13 +13,7 @@
 # Define the default flow function for computing maximum flow.
 default_flow_func = preflow_push
 # Functions that don't support cutoff for minimum cut computations.
-flow_funcs = [
-    boykov_kolmogorov,
-    dinitz,
-    edmonds_karp,
-    preflow_push,
-    shortest_augmenting_path,
-]
+flow_funcs_without_cutoff = [preflow_push]
```

Comment:
Rather than introduce a new variable, I'd vote to modify the check to check against the function directly (see below).

If there's a strong desire to keep the new variable, we should at least make it private so it's clear to users that it's intended for internal use only, i.e. `_flow_funcs_without_cutoff`.

Reply:
I agree. I think it's easier to just test against that function. When I started fixing this I thought there was more than 1 function that didn't allow cutoff.